cristin-commons/src/main/java/no/unit/nva/cristin/common/client/CristinAuthenticator.java

```diff
@@ -42,10 +43,11 @@ protected PasswordAuthentication getPasswordAuthentication() {
      */
     public static HttpClient getHttpClient() {
         return HttpClient.newBuilder()
-                .followRedirects(HttpClient.Redirect.ALWAYS)
-                .authenticator(getBasicAuthenticator())
-                .connectTimeout(Duration.ofSeconds(30))
-                .build();
+                   .followRedirects(HttpClient.Redirect.ALWAYS)
```

Comment:
Why not use the HttpClientProvider here?

Reply:
Ahh, the authenticator. Did you consider adding support for authenticator in HttpClientProvider?

Reply:
we use `.authenticator` which is not present in the default client method

Reply:
Did not consider adding support in HttpClientProvider since it is only used one place, and that is here.